hs_collection_resource/tests/my_res_file.txt

```diff
@@ -0,0 +1 @@
+I am a poor txt file for testing purpose. Don't remove me!
```

Comment:
@zhiyuli I don't see a reason to keep this empty test holder in place now. I think it can be removed unless there is a reason I don't see now.